gpu_bdb/benchmark_runner/slurm/rapids-gpu-bdb-cuda11.2.yml

```diff
@@ -0,0 +1,31 @@
+channels:
```

Comment:
is there no way we can just use https://github.com/rapidsai/gpu-bdb/blob/main/conda/rapids-gpu-bdb-dask-sql.yml instead of duplicating the file?

relatedly, we're missing cupy here, I think.

And, might we want to be using Dask/Distributed/SQL from source, instead of released versions on conda?

Reply:
I think the only difference we'd want to maintain between the two files is the fact that we don't want to install dask-sql from conda in this one. It installs maven instead which is a pre-requisite for building from source.

Reply:
I'd be fine pushing maven into the main env file (to enable dask-sql source builds), and getting rid of these now duplicate env files.

Don't want anybody to have to check the versions here ever again =)